code/modules/mob/new_player/new_player.dm

```diff
@@ -244,6 +244,10 @@
 	RoleAuthority.equip_role(character, RoleAuthority.roles_for_mode[rank], T)
 	EquipCustomItems(character)
 
+	if(security_level > SEC_LEVEL_BLUE || EvacuationAuthority.evac_status)
+		to_chat(character, SPAN_HIGHDANGER("As you stagger out of hypersleep, the sleep bay blares: '[EvacuationAuthority.evac_status ? "VESSEL UNDERGOING EVACUATION PROCEDURES, SELF DEFENSE KIT PROVIDED" : "VESSEL IN HEIGHTENED ALERT STATUS, SELF DEFENSE KIT PROVIDED"]'."))
+		character.put_in_hands(new /obj/item/storage/box/kit/cryo_self_defense(character.loc))
```

Comment:
what if they start with two custom items in their hands? make it drop to their loc

Reply:
> what if they start with two custom items in their hands? make it drop to their loc

this is rly embarrassing you kinda got mogged by the code here stan